### PR TITLE
Acceptance process delegation for most items + exceptions

### PR DESCRIPTION
**Pull request by taylor**
This adds the approval process section which 
- delegates most responsibility for accepting changes to the community as outlined in the CONTRIBUTING guide
- exception list of items requiring additional approval

ref: 
- discussion https://github.com/cncf/cnf-wg/discussions/130
- approval process issue #74

### CODE REVIEW

**Comment by taylor** [comment]
@electrocucaracha the link checks are going against master instead of the branch with the change.  anything to do?

cc: @iawells

File: GOVERNANCE.md [hunk 1]
```diff
@@ -107,3 +107,33 @@ If insufficient members resign their roles, the positions held by members associ
...
+### Changes requiring additional approval
+
+Items in the Additional Approval Items list require further discussion and approval before being accepted. The decision making process for items in this list is TBD.
+
+
+
+#### Additional Approval Items
+1. This list
+1. [Charter](https://github.com/cncf/cnf-wg/blob/master/charter.md)
+1. [Governance](https://github.com/cncf/cnf-wg/blob/master/governance.md)
```
Comment (electrocucaracha) [inline GOVERNANCE.md]: Can you try with upper case? It seems to be case sensitive.
```suggestion
1. [Governance](https://github.com/cncf/cnf-wg/blob/master/GOVERNANCE.md)
```

Comment (iawells) [reply GOVERNANCE.md]: They should be relative links, as I mentioned in another pull request. A branch of fork needs to be consistent with itself, not with master in the main repo.

Comment (iawells) [reply GOVERNANCE.md]: (It needed fixing before; you just happen to have tripped over why.)

File: GOVERNANCE.md [hunk 2]
```diff
@@ -107,3 +107,33 @@ If insufficient members resign their roles, the positions held by members associ
...
+1. [Code of Conduct](https://github.com/cncf/cnf-wg/blob/master/code-of-conduct.md)
+1. CNF WG "Approved" Best Practices
+    - This would be Best Practices which the CNF WG promotes in an approved list. Eg. Baseline Best Practices
```
Comment (sishbi) [inline GOVERNANCE.md:130]: Should there be a 'stub' file for the approved list, such as 'approved-best-practices.md' ?

**Review by sishbi — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,6 +66,28 @@ Pull Requests are always welcome, even for small fixes like typos.
 
 Reviews and comments on open [Pull Requests](https://github.com/cncf/cnf-wg/pulls) are also appreciated. 
 
+##### Steps for creating a new PR:
+- Once you have made your change or added new content
+- Ensure you are up to date with the `main` branch of the cnf-wg repository
+- Open a new [Pull Request](https://github.com/cncf/cnf-wg/pulls)
+- Tag at least 5 reviewers
```
Comment (xmulligan) [inline CONTRIBUTING.md]: Where do people find those reviewers? If someone new is coming to the repo, they won't know who to tag

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -66,6 +66,28 @@ Pull Requests are always welcome, even for small fixes like typos.
...
+
+#### Steps to accept a PR
+1. Reviewers will review the PR and give their feedback: approve, request change, comment w/o approval
```
Comment (xmulligan) [inline CONTRIBUTING.md:81]: Who are "Reviewers"? Who can be one?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -66,6 +66,28 @@ Pull Requests are always welcome, even for small fixes like typos.
...
+Anyone with merge access can merge the PR after the item has been approved.
+
+
+### Acceptance of Contributions
+
+Minor changes:
+- Spelling, grammar, and adding new interested parties require 1 reviewer
+
+Majority of other changes:
+- Other changes with exception to those listed in the [Governance Additional Approval Items list](GOVERNANCE.md#additional-approval-items) may accepted after approval of 5 community reviewers.
```
Comment (xmulligan) [inline CONTRIBUTING.md]: Who a "community reviewer" is needs to be defined somewhere. What about adding a requirement that they come from at least 3 different companies?

File: GOVERNANCE.md [hunk 6]
```diff
@@ -107,3 +107,33 @@ If insufficient members resign their roles, the positions held by members associ
 #### Effect on representation
 
 After any election, the results may indicate that a particular entity is overrepresented for a role, having won more seats than the limit.  In this event, the maximum number of the members belonging to that entity shall be calculated, and members belonging to the entity having the least votes will be excluded from the role until the entity is fairly represented.  Where there is a tie, a coin toss or other random selection mechanism shall be used to choose who is allocated the seat.
+
+
+## Content Approval Process
+
+To encourage contributions the CNF WG has a more relaxed stance on changes and delegates the acceptance for most items to the community as outlined in CONTRIBUTING.md. The small list of exceptions are listed below which require a more thorough decision making process to acheive our desired outcome of fair representation and technical excellence.
```
Comment (xmulligan) [inline GOVERNANCE.md]:
```suggestion
To encourage contributions, the CNF WG has a more relaxed stance on changes and delegates the acceptance for most items to the community as outlined in CONTRIBUTING.md. The small list of exceptions are listed below which require a more thorough decision making process to achieve our desired outcome of fair representation and technical excellence.
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (jeffsaelens) [reply CONTRIBUTING.md]: Could we add an LFR (Looking For Reviewers) label?

**Review by Smitholi67 — APPROVED** [review]
Looks good to me.  Agree with the comment by @sishbi that the best practices baseline should sit in a separate file.  Thinking forward it might be good to maintain different versions of these best practices as they continue to mature.  This would be useful if the best practices are to be used in a badging process down the road.

File: CONTRIBUTING.md (hunk 3 above)
Comment (taylor) [reply CONTRIBUTING.md]: @xmulligan ideally we could use github teams to add all reviewers to the team. Then you can tag the whole group like: @cncf/cnf-wg-reviewers. Unfortunately teams only allow org members and not "outside collaborators".  

Someone new can click the review button on the PR and select folks, but we do have people listed who may not be active or just happen to have access via the org level permissions.  
image

One option is to create a new org for all of the CNF initiatives. We thought about this in the past, but only had cnf-testbed at the time. Now there are 4.  If we did this we could create teams and add everyone who is participating to the new "CNCF CNF" org.

An alternative approach to deal with the github limitations are to to create a file in the repo we can add the github user handle (and probably name) for anyone that would like to be a reviewer. K8s and many projects uses a file called OWNERS to list reviewers and other roles for tagging.

From there someone can manually look at the file and pick some user names to tag (with the reviewers button or via comment) or we can add a bot like many projects have which supports using an owners file.

Many projects and communities use automation with something like an OWNERS file including the K8s community.  They use prow and the bot has a ton of features -- https://github.com/kubernetes/test-infra/blob/master/prow/README.md.  We can probably go with something simpler if we want to automate things.

Github supports some automation via a CODEOWNERS file, but it has limitations such as automatically notifying everyone that is listed.  That could get spammy of course if everyone is notified for every new PR to review vs a subset. See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

**TL;DR**
- Couple of options
    - Create a file with a list of the reviewers
    - Create new CNCF CNF org (eg. cncf-cnf) and use org teams
- Decide on manual or automation

Comment (taylor) [reply CONTRIBUTING.md]: @jeffsaelens yes for a label.  Who will look for the label though?  If we automate or its manually noticed and an action to choose reviewers happens when the label is there we still need a list of reviewers somewhere.

File: CONTRIBUTING.md (hunk 4 above)
Comment (taylor) [reply CONTRIBUTING.md:81]: @xmulligan any community member can be a reviewer.

Maybe it should say Community Reviewers or Community Member?

We can grant github read access to the repo to allow the reviewer to be tagged in a PR in the reviews section.

File: CONTRIBUTING.md (hunk 5 above)
Comment (taylor) [reply CONTRIBUTING.md]: Yes for a definition being somewhere.  I was thinking anyone in the community can say they would like to review.  So this is not a select set for the default reviews.  Open to anyone in the community. We can add other types of reviewers later if needed.

Not sure that its necessary at this time to limit reviews from the same org/company yet for the items not in the Additional Approval Items list.  We could modify the CONTRIBUTING guide when we see a problem or after further thought. This path was to keep it simple for the defaults.

Specifically for non-controversial items we want to lower the barrier for making contributions.  If the barrier is too low we adjust later. If something needs more discussion, eg. its something that is more "controversial", then maybe it gets added to the other list.

File: GOVERNANCE.md (hunk 2 above)
Comment (taylor) [reply GOVERNANCE.md:130]: @sishbi possibly.  it does not seem needed for this PR, IMO though.  We do have https://github.com/cncf/cnf-wg/blob/master/doc/best_cnf_dev.md which somewhat is communicating a baseline set of practices for cnf developers.  Do you think its necessary for this PR or could it be addressed later?

Comment (taylor) [reply GOVERNANCE.md:130]: I do think, the definition of approved best practices and/or the name in the list could use some some clarity.

If its not apparent there is not a restriction on proposing a best practice.  There is not a restriction on accepting a PR to add a proposal to the repository. The restriction is in communicating that the CNF WG now advocates the proposed / draft best practice should be adopted. This would also tie into any type of conformance program. We want to make sure that best practices that are included in a conformance program go through additional approval.

Comment (sishbi) [reply GOVERNANCE.md:130]: @taylor Maybe the fix is to add a link to that doc as I missed it when I first read through the wiki
Is that the current approved list? Or the current draft list?

**Comment by iawells** [comment]
> the link checks are going against master instead of the branch with the change. anything to do?

Could have sworn I said this earlier but maybe it was another PR: the link checks should be relative links (so that they point to the PR's documents) and never absolute links to both the main repo and the main branch.   Broken links is one of the reasons why, but of course another is you can't easily review a document set for consistency when its crosslinks take you to inconsistent versions.

While we do want someone to go through the lot and deal with it, for now I suggest we weed them out as we find them, which means this wants fixing up as you do this PR.

File: CONTRIBUTING.md (hunk 3 above)
Comment (iawells) [reply CONTRIBUTING.md]: To get us started (until we find a nice technical solution or something else) I would basically say 'tag at least 5 reviewers OR tag a co-chair who will find reviewers for you'.  It doesn't have to be perfect just yet, only functional.

**Comment by iawells** [comment]
Speaking to intentions a moment:

- first, we need people to review and merge things (this is all a bit woolly at the moment and we seem to have a bit of merge hesitancy; this PR makes it clear when merges should happen)
- then, we need to develop use cases and best practices (so we need a procedure that works specifically for them right now)
- finally, we need to set a baseline for release (at which point we'll need an approval process, and we'll need to ensure people don't modify old baselines, and so on)

I like the way this is put together, because it codifies the bits that we _need_ without getting too in-depth in the things we haven't reached just yet.  That works for me because I think we'll learn things on the journey - rather than having a stab at the later bits right now, we will write them up a bit closer to when we need them and we understand how we work a little bit more clearly.

File: CONTRIBUTING.md (hunk 4 above)
Comment (iawells) [reply CONTRIBUTING.md:81]: I see potential reviewers as any community member for the moment.  We can get more specific in the future, but for the moment this lets us get moving (and is, in practice, writing down what we're already doing).

**Review by iawells — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 5 above)
Comment (jeffsaelens) [reply CONTRIBUTING.md]: I think the last paragraph is what's important. For basic house keeping, we should make things simple to keep things moving smoothly. Company separation for grammar/spelling fixes, or adding events to a list somewhere etc... don't need any special protection. 

As we begin to produce best practices, and things that have impact, we can tighten up what a role to approve merges would look like.

File: CONTRIBUTING.md (hunk 3 above)
Comment (iawells) [reply CONTRIBUTING.md]: Indeed: if we make a team of 'standard reviewers' it can be their first responsibility to spread the joy to others (which works slightly better than a label would, imo)

Comment (ildikov) [reply CONTRIBUTING.md]: Is this step a must?

As GH sends notification emails, I wonder if the notification about tagging will be lost or rather just a duplicate of the notification about the new PR. Would this be better as a step to take if the PR sits unreviewed for longer than X?

File: CONTRIBUTING.md (hunk 4 above)
Comment (ildikov) [reply CONTRIBUTING.md:81]: If we have a list of identified roles I think it's better to document that somewhere and refer to that list where needed as opposed to trying to create descriptive terms like 'community reviewer'. As everyone in the community should be able and encouraged to review, saying 'community reviewer' creates a confusing distinction as I don't think we will have non-community reviewers.

File: CONTRIBUTING.md (hunk 5 above)
Comment (ildikov) [inline CONTRIBUTING.md]:
```suggestion
- Other changes with exception to those listed in the [Governance Additional Approval Items list](GOVERNANCE.md#additional-approval-items) may be accepted after approval of 5 community reviewers.
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (taylor) [reply CONTRIBUTING.md]: @ildikov do you mean comment tagging (team or individuals) or something else

I was taking @iawells comment
>tag at least 5 reviewers OR tag a co-chair who will find reviewers for you'
to mean typing the usernames into the choose reviewers box on the top right of a pull request re: screenshot https://github.com/cncf/cnf-wg/pull/131#discussion_r610820564 with _"Request up to 15 reviewers"_

Comment (taylor) [reply CONTRIBUTING.md]: @iawells @ildikov see latest update to CONTRIBUTING guide:
- Choose at least 5 reviewers OR choose a co-chair who will find reviewers for you

File: CONTRIBUTING.md (hunk 4 above)
Comment (taylor) [reply CONTRIBUTING.md:81]: @iawells @ildikov see [latest update to contributing guide](https://github.com/cncf/cnf-wg/blob/minimal-approval-process-with-exception-list/CONTRIBUTING.md#acceptance-of-contributions)

Reviewers - Any community member
- Everyone in the community is able to and encouraged to do reviews

File: CONTRIBUTING.md (hunk 5 above)
Comment (taylor) [reply CONTRIBUTING.md]: @xmulligan see [latest update to contributing guide](https://github.com/cncf/cnf-wg/blob/minimal-approval-process-with-exception-list/CONTRIBUTING.md#acceptance-of-contributions)

Reviewers - Any community member
- Everyone in the community is able to and encouraged to do reviews

File: GOVERNANCE.md (hunk 2 above)
Comment (taylor) [reply GOVERNANCE.md:130]: @sishbi see [latest update to GOVERNANCE.md](https://github.com/cncf/cnf-wg/blob/minimal-approval-process-with-exception-list/GOVERNANCE.md#additional-approval-items) with a link to the dev doc

We have no draft list yet.  That document was created to give some structure for a potential list of best practices.  It could be the "approved" list or a list of all of them including non-approved.

File: GOVERNANCE.md (hunk 6 above)
Comment (taylor) [inline GOVERNANCE.md]:
```suggestion
To encourage contributions the CNF WG has a more relaxed stance on changes and delegates the acceptance for most items to the community as outlined in CONTRIBUTING.md. The small list of exceptions are listed below which require a more thorough decision making process to achieve our desired outcome of fair representation and technical excellence.
```

**Comment by taylor** [comment]
@iawells links should now be relative

https://github.com/cncf/cnf-wg/pull/131#issuecomment-816896649

**Review by jeffsaelens — APPROVED** [review]
changes look good.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -66,6 +66,31 @@ Pull Requests are always welcome, even for small fixes like typos.
 
 Reviews and comments on open [Pull Requests](https://github.com/cncf/cnf-wg/pulls) are also appreciated. 
 
+##### Steps for creating a new PR:
+- Once you have made your change or added new content
+- Ensure you are up to date with the `main` branch of the cnf-wg repository
+- Open a new [Pull Request](https://github.com/cncf/cnf-wg/pulls)
+- Choose at least 5 reviewers OR choose a co-chair who will find reviewers for you
```
Comment (electrocucaracha) [inline CONTRIBUTING.md]: Given the current co-chairs are listed in the governance document maybe they can be referenced
```suggestion
- Choose at least 5 reviewers OR choose a [co-chair](https://github.com/cncf/cnf-wg/blob/main/GOVERNANCE.md#chairs) who will find reviewers for you
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -66,6 +66,31 @@ Pull Requests are always welcome, even for small fixes like typos.
...
+
+#### Steps to accept a PR
+1. Reviewers will review the PR and give their feedback: approve, request change, comment w/o approval
+1. After required number of approvals from reviewers is reached the PR may be merged
+
+Anyone with merge access can merge the PR after the item has been approved.
+
+
+### Acceptance of Contributions
```
Comment (electrocucaracha) [inline CONTRIBUTING.md]:
```suggestion
### Acceptance criteria for Contributions
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -66,6 +66,31 @@ Pull Requests are always welcome, even for small fixes like typos.
...
+
+Reviewers - Any community member
+- Everyone in the community is able to and encouraged to do reviews
+
+Minor changes:
+- Spelling, grammar, and adding new interested parties require 1 reviewer
```
Comment (electrocucaracha) [inline CONTRIBUTING.md:94]: Maybe we can apply [lazy consensus](https://community.apache.org/committers/lazyConsensus.html) for those minor changes

File: GOVERNANCE.md [hunk 10]
```diff
@@ -107,3 +107,32 @@ If insufficient members resign their roles, the positions held by members associ
...
+
+
+#### Additional Approval Items
+1. This list
+1. [Charter](charter.md)
+1. [Governance](GOVERNANCE.md)
+1. [Code of Conduct](code-of-conduct.md)
+1. CNF WG "Approved" Best Practices
+    - This would be Best Practices which the CNF WG promotes in an approved list. Eg. Baseline Best Practices
+    - Proposing a best practice is okay with the default process. Marking it as accepted and listing it as a CNF WG approved Best Practice requires additional review and approval
```
Comment (agentpoyo) [inline GOVERNANCE.md:133]: `Proposing a best practice is welcome and encouraged with the default process.`

Reads better than using okay.

**Review by agentpoyo — CHANGES_REQUESTED** [review]

File: GOVERNANCE.md (hunk 10 above)
Comment (agentpoyo) [inline GOVERNANCE.md]:
```suggestion
    - Proposing a best practice is encouraged with the default process. Marking it as accepted and listing it as a CNF WG approved Best Practice requires additional review and approval.
```

**Review by agentpoyo — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -66,6 +66,31 @@ Pull Requests are always welcome, even for small fixes like typos.
...
+
+Majority of other changes:
+- Other changes with exception to those listed in the [Governance Additional Approval Items list](GOVERNANCE.md#additional-approval-items) may be accepted after approval of 5 community reviewers.
```
Comment (pgoyal01) [inline CONTRIBUTING.md]: Should there be a requirement on the diversity of reviewers (such as not more than 2 from the same company, at least X from each company industry, not from the PR author company, etc,

File: CONTRIBUTING.md (hunk 9 above)
Comment (pgoyal01) [reply CONTRIBUTING.md:94]: Should there be a restriction that the approver is not from same company as the author

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -66,6 +66,31 @@ Pull Requests are always welcome, even for small fixes like typos.
...
 
+##### Steps for creating a new PR:
+- Once you have made your change or added new content
+- Ensure you are up to date with the `main` branch of the cnf-wg repository
+- Open a new [Pull Request](https://github.com/cncf/cnf-wg/pulls)
+- Choose at least 5 reviewers OR choose a co-chair who will find reviewers for you
+
+#### Steps to accept a PR
+1. Reviewers will review the PR and give their feedback: approve, request change, comment w/o approval
+1. After required number of approvals from reviewers is reached the PR may be merged
```
Comment (pgoyal01) [inline CONTRIBUTING.md:82]: Should there be an add that the issues/questions raised have been addressed? Co-chair should ensure prior to merge -- co-chairs responsible for merge or insist a co-chair approval.

Comment (pgoyal01) [reply CONTRIBUTING.md:82]: Are approvals to be ignored that were made prior to non-trivial changes made to the PR?

File: CONTRIBUTING.md (hunk 9 above)
Comment (taylor) [reply CONTRIBUTING.md:94]: @pgoyal01 that might be a good idea. @xmulligan had a related suggestion about reviewers.  

For now I suggest we leave it more open, then modify if we see it as a problem.  The CONTRIBUTING.md can be updated based on the criteria we list here.  Eg. 5 community reviewers.

**Review by agentpoyo — APPROVED** [review]
LGTM

**Review by electrocucaracha — APPROVED** [review]

**Review by rannyh — APPROVED** [review]
LGTM

**Review by pgoyal01 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -64,11 +64,37 @@ Pull Requests are always welcome, even for small fixes like typos.
 
 Reviews and comments on open [Pull Requests](https://github.com/cncf/cnf-wg/pulls) are also appreciated.
 
+#### Steps for creating a new PR:
+
+- Once you have made your change or added new content
+- Ensure you are up to date with the `main` branch of the cnf-wg repository
+- Open a new [Pull Request](https://github.com/cncf/cnf-wg/pulls)
+- Choose at least 5 reviewers OR choose a [co-chair](https://github.com/cncf/cnf-wg/blob/main/GOVERNANCE.md#chairs) who will find reviewers for you
```
Comment (lixuna) [inline CONTRIBUTING.md]: suggestion to change link from (https://github.com/cncf/cnf-wg/blob/main/GOVERNANCE.md#chairs) to (GOVERNANCE.md#chairs)

File: GOVERNANCE.md [hunk 14]
```diff
@@ -105,3 +105,31 @@ If insufficient members resign their roles, the positions held by members associ
 ### Effect on representation
 
 After any election, the results may indicate that a particular entity is overrepresented for a role, having won more seats than the limit.  In this event, the maximum number of the members belonging to that entity shall be calculated, and members belonging to the entity having the least votes will be excluded from the role until the entity is fairly represented.  Where there is a tie, a coin toss or other random selection mechanism shall be used to choose who is allocated the seat.
+
+
+## Content Approval Process
+
+To encourage contributions, the CNF WG has a more relaxed stance on changes and delegates the acceptance for most items to the community as outlined in CONTRIBUTING.md. The small list of exceptions are listed below which require a more thorough decision making process to achieve our desired outcome of fair representation and technical excellence.
```
Comment (lixuna) [inline GOVERNANCE.md]:
```suggestion
To encourage contributions, the CNF WG has a more relaxed stance on changes and delegates the acceptance for most items to the community as outlined in [CONTRIBUTING.md](CONTRIBUTING.md). The small list of exceptions are listed below which require a more thorough decision making process to achieve our desired outcome of fair representation and technical excellence.
```

File: GOVERNANCE.md [hunk 15]
```diff
@@ -105,3 +105,31 @@ If insufficient members resign their roles, the positions held by members associ
...
+
+### Delegation of default changes to community
+
+The process for accepting changes to content in the CNF WG repository (including adding new content) for any item not in the [Additional Approval Items list](#DD) below will follow the acceptance and approval process outlined in the [CONTRIBUTING guide](https://github.com/cncf/cnf-wg/blob/master/CONTRIBUTING.md).
```
Comment (lixuna) [inline GOVERNANCE.md]:
```suggestion
The process for accepting changes to content in the CNF WG repository (including adding new content) for any item not in the [Additional Approval Items list](#DD) below will follow the acceptance and approval process outlined in the [CONTRIBUTING guide](CONTRIBUTING.md).
```

File: CONTRIBUTING.md (hunk 13 above)
Comment (lixuna) [inline CONTRIBUTING.md]:
```suggestion
- Choose at least 5 reviewers OR choose a [co-chair](GOVERNANCE.md#chairs) who will find reviewers for you
```

**Comment by taylor** [comment]
@lixuna all your items have been addressed. Please re-review

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -64,11 +64,47 @@ Pull Requests are always welcome, even for small fixes like typos.
...
+Reviewers - Any community member
+- Everyone in the community is able to and encouraged to do reviews
+
+Minor changes:
+- Spelling, grammar, and adding new interested parties require 1 reviewer
+
+Majority of other changes:
+- Other changes with exception to those listed in the [Governance Additional Approval Items list](GOVERNANCE.md#additional-approval-items) may be accepted after approval of 5 community reviewers.
+
+<<<<<<< HEAD
```
Comment (iawells) [inline CONTRIBUTING.md]: oopsy

**Review by iawells — APPROVED** [review]

File: CONTRIBUTING.md (hunk 16 above)
Comment (taylor) [reply CONTRIBUTING.md]: oops

File: CONTRIBUTING.md (hunk 12 above)
Comment (taylor) [reply CONTRIBUTING.md:82]: @pgoyal01 I try to re-request reviews for non-trivial changes on PRs.

Comment (taylor) [reply CONTRIBUTING.md:82]: @pgoyal01 I'm trying to go through and address all comments in the PR.

If you want it in the CONTRIBUTING.md then please open a new issue and/or PR.

Also remember we can always go back and open a new PR to address anything we consider a mistake or now want to see differently.

File: CONTRIBUTING.md (hunk 11 above)
Comment (taylor) [reply CONTRIBUTING.md]: @pgoyal01 this could be brought up as a new issue, discussion, or PR.

**Review by lixuna — APPROVED** [review]
lgtm

**Comment by jeffsaelens** [comment]
There is a ton going on here. There have been previous approvals, and 3 current ones. Could we get some fresh eyes on this? We may be at the point where we want to merge this, and then cut a new PR for additional modifications.

**Comment by taylor** [comment]
@jeffsaelens unfortunately github does not show approvers in the history unless they have specific permission for the repo.  That's why you don't find @electrocucaracha in the history and yet has has approved in the reviewers area (✔️ ).


Reviewers from today who have approved: @iawells @lixuna @agentpoyo @rannyh @pgoyal01 @electrocucaracha 

The past reviewers are: @sishbi @Smitholi67 

I'll re-request for the past reviewers.  Anyone else not listed above would be new.

**Review by iawells — APPROVED** [review]
Still good with this in its current state.

**Review by rabi-abdel — APPROVED** [review]
It will be good to define who can be considered a community member (it could be simply any one).

**Review by ildikov — APPROVED** [review]

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -64,11 +64,45 @@ Pull Requests are always welcome, even for small fixes like typos.
 
 Reviews and comments on open [Pull Requests](https://github.com/cncf/cnf-wg/pulls) are also appreciated.
 
+#### Acceptance of PRs
+
+- Spelling, grammar, and new interested parties require 1 reviewer
+- Other changes with exception to those listed in the [Governance Additional Approval Items list](GOVERNANCE.md#additional-approval-items) may accepted after approval of 5 community reviewers.
```
Comment (xmulligan) [inline CONTRIBUTING.md]:
```suggestion
#### Acceptance of PRs for merging

- Spelling, grammar, and new interested parties require 1 apporval for merging
- Other changes with exception to those listed in the [Governance Additional Approval Items list](GOVERNANCE.md#additional-approval-items) may merged after approval of 5 community reviewers.
```

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -64,11 +64,45 @@ Pull Requests are always welcome, even for small fixes like typos.
...
+
+- Everyone in the community is able to and encouraged to do reviews
+
+Minor changes:
+
+- Spelling, grammar, and adding new interested parties require 1 reviewer
+
+Majority of other changes:
+
+- Other changes with exception to those listed in the [Governance Additional Approval Items list](GOVERNANCE.md#additional-approval-items) may be accepted after approval of 5 community reviewers.
```
Comment (xmulligan) [inline CONTRIBUTING.md]:
```suggestion
- Other changes with exception to those listed in the [Governance Additional Approval Items list](GOVERNANCE.md#additional-approval-items) may be merged after approval of 5 community reviewers.
```

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -64,11 +64,45 @@ Pull Requests are always welcome, even for small fixes like typos.
...
+1. Reviewers will review the PR and give their feedback: approve, request change, comment w/o approval
+1. After required number of approvals from reviewers is reached the PR may be merged
+
+Anyone with merge access can merge the PR after the item has been approved.
+
+### Acceptance criteria for Contributions
+
+Reviewers - Any community member
+
+- Everyone in the community is able to and encouraged to do reviews
```
Comment (xmulligan) [inline CONTRIBUTING.md]:
```suggestion
- Everyone in the community is able to and encouraged to do reviews. Anyone can be a part of the CNF WG community.
```

**Review by xmulligan — APPROVED** [review]

**Review by jeffsaelens — APPROVED** [review]

**Review by sishbi — COMMENTED** [review]
Some minor edits in PR #155
